src/components/HelperErrorText/HelperErrorText.tsx

```diff
@@ -44,11 +40,10 @@ export const HelperErrorText = chakra(
       ref?
     ) => {
       // Only announce the text in the invalid state.
-      const announceAriaLive = isInvalid;
       const styles = useStyleConfig("HelperErrorText", { isInvalid });
       const props = {
         "aria-atomic": ariaAtomic,
-        "aria-live": announceAriaLive ? ariaLive : "off",
+        "aria-live": ariaLive ? ariaLive : undefined,
```

Comment:
Do we need this ternary if the default parameter is already undefined? I think this could just be `"aria-live": ariaLive`.

Reply:
Good catch.  That change still allows the component to function properly.